mitxgraders/formulagrader/formulagrader.py

```diff
@@ -420,6 +420,11 @@ class FormulaGrader(ItemGrader):
                     should return True, False, 'partial', or a dictionary with required key
                     'grade_decimal' and optional key 'msg'. Comparer messages are ignored
                     when comparison succeeds (result['ok'] is True).
+
+        instructor_vars ([str]): A list of variable/constant names that cannot be used by
+            students. This can be useful in constructing DependentSampler expressions or
+            blacklisting constants. Note that this list is not validated against the list
```

Comment:
Idea: could we just make `blacklist` accept (func names, var names, constant names) instead of just function names?

(I haven't though about this too much from an implementation point of view.)

Reply:
I don't want to do that, because it's possible to have functions and variables with the same name, and blacklist just takes in a list of strings. It would be impossible to distinguish between them, no?

Reply:
> because it's possible to have functions and variables with the same name

Good point. I wish we had called it `blacklist_funcs` and `blacklist_vars`.

Anyway, `instructor_vars` is the same goal as blacklist, just for vars instead of funcs. What do you think of calling it `blacklist_vars`?

The only disadvantage is that its for vars and consts, but `instructor_vars` has that disadvantage, too.

Reply:
Actually, insturctor_vars is good. I'm just gonna merge.

Reply:
Hah, did you decide that it wasn't quite the same thing as a blacklist?